test/fixtures/copy_license/no_licenses/lib/non-utf8

```diff
@@ -1 +0,0 @@
-�
```

Comment:
Was this file actually removed? Or is GitHub just not picking up the `test/fixtures` -> `test_fixtures` rename?